### DISCUSSION TITLE
Upgraded to latest from V5, no access to admin page

### DISCUSSION THREAD

**Discussion by cfc62** [chuot/rdio-scanner#93] [General]
Upgraded from v5 (somewhat) successfully, but while I was able to see the admin page once, any subsequent attempts to view the admin page are denied. 

I never changed the password (never had the chance!), and I also re-ran through the upgrade procedure and I am still locked out. 

So, successful in that the process seems to be running fine, but unsuccessful in that I can't update the configuration at all, which is frustrating.

**Comment by cfc62** [comment]
Sorry, should have included version: 6.1.3 which I see has an admin pw fix, perhaps that fix broke something else...

**Comment by canuckcam** [comment]
Did you get a 504 Gateway error even if you reset the password? I upgraded from 5.2.9 to 6.1.3 on Windows and got that error. For me, I had -listen on port :80/admin so it showed me the admin login screen, but trying the same in on localhost:3000/admin worked! A reboot (or logging in on port 3000 once) fixed the issue, and no longer able to get to the page on port 3000.

**Comment by cfc62** [comment]
I'm seeing the admin login page at the right location and it's correctly asking for the password, it's just denying every password entry.

I haven't tried a reboot (it's a server serving multiple functions) but I may try that later.

**Comment by canuckcam** [comment]
Yes, that's exactly what happened to me. I got the admin login page and with the correct password it wouldn't return a wrong password error but rather the 504 error. BUT if I went to http://host:3000/admin even though the proper listening port is 80, it would display the admin login page and the password worked. After this happened, I didn't click around but rebooted the computer and I could login again on port 80 as normal.

I guess restarting is now a bit more disruptive than just restarting a docker container. The alternative would be a small Linux VM or LXC?
